docs/contributing/semgrep-core-contributing.md

```diff
@@ -146,6 +146,85 @@ code .
 
 ## Testing performance
 
+### Exploring results from a slow run of Semgrep
+
+#### Interpreting the result object
+
+For full timing information, run Semgrep with `--time` and `--json`. In addition, you will want to `time` the entire command to get the true wall time. The `--json` argument produces a large amount of output, so redirecting the output to a file with `-o` is recommended. The full command would look sometthing like:
+
+<pre class="language-bash"><code>time semgrep --config=auto --time --json -o result.json <span className="placeholder">PATH/TO/SRC</span></code></pre>
+
+Substitute the optional placeholder <code><span className="placeholder">PATH/TO/SRC</span></code> with the path to your source code.
+
+Here is an example result object.
+
+```JSON
+{
+  "errors": [],
+  "results": [],
+  "time": {
+    "profiling_times": {
+      "config_time": 0.10301780700683594,
+      "core_time": 0.0883018970489502,
+      "ignores_time": 2.7894973754882812e-05,
+      "total_time": 0.1915416717529297
+    },
+    "rule_parse_info": [
+      0.0011630058288574219
+    ],
+    "rules": [
+      {
+        "id": "Users.emma.workspace.testing.error_configs.use-sys-exit"
+      }
+    ],
+    "targets": [
+      {
+        "match_times": [
+          0
+        ],
+        "num_bytes": 444,
+        "parse_times": [
+          0
+        ],
+        "path": "/Users/emma/workspace/testing/situations_test/no_token_location.py",
+        "run_times": [
+          0.0006248950958251953
+        ]
+      }
+    ],
+    "total_bytes": 444
+  }
+}
+```
+
+All the information about timing is contained under `time`.
+
+The first section is `profiling_times`. This contains wall time durations of various steps we consider interesting: getting the rule config files (`config_time`), running the main engine (`core_time`), and processing the ignores (`ignores_time`). The `total_time` field represents the sum of these steps.
```

Comment:
```suggestion
The first section is `profiling_times`. This contains wall time durations of various relevant steps:
* Getting the rule config files (`config_time`)
* Running the main engine (`core_time`)
* Processing the ignores (`ignores_time`) 

The `total_time` field represents the sum of these steps.
```

Reply:
Follow up: I'll make a separate follow up PR to convert these definitions into a list, either a <dl> description list or <ul>.